homeassistant/components/remootio/device_trigger.py

```diff
@@ -0,0 +1,109 @@
+"""Provides device automations for Remootio."""
```

Comment:
Can device triggers be split off into a separate PR?

Reply:
I can split it, but is this really needed?

Reply:
Yes, as per our PR guidelines it's needed.

Reply:
Oh, OK. I will split it off. Sorry!